azure-pipelines.yml

```diff
@@ -21,18 +17,77 @@ pr:
 variables:
   OFFICIAL_REPO: 'pyqtgraph/pyqtgraph'
 
-jobs:
+stages:
+- stage: "pre_test"
+  jobs:
+  - job: check_diff_size
+    pool:
+      vmImage: 'Ubuntu 16.04'
+    steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: 3.7
+    - bash: |
+        mkdir ~/repo-clone && cd ~/repo-clone &&
+        git init && git remote add -t $(Build.SourceBranch) origin $(Build.Repository.Uri) &&
+        git fetch origin $(Build.SourceBranch) &&
+        git checkout -qf FETCH_HEAD &&
+        git tag azure-merge-target &&
+        git gc --aggressive &&
+        TARGET_SIZE=`du -s . | sed -e "s/\t.*//"` &&
+        git pull origin $(Build.SourceBranch) &&
+        git gc --aggressive &&
+        MERGE_SIZE=`du -s . | sed -e "s/\t.*//"` &&
+        if [ "${MERGE_SIZE}" != "${TARGET_SIZE}" ]; then
+          SIZE_DIFF=`expr \( ${MERGE_SIZE} - ${TARGET_SIZE} \)`;
+        else
+          SIZE_DIFF=0;
+        fi;
+        echo -e "Estimated content size difference = ${SIZE_DIFF} kB" &&
+        test ${SIZE_DIFF} -lt 100;
+        # python setup.py mergesize
```

Comment:
Why have the merge-size check implemented both here and in setupHelpers?

Reply:
right, it was implemented in setupHelpers; but before this gets merged, it should really be tested via both methods... not sure if I should have the CI run the bash or setupHelper method